src/lib.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+//!  This example shows you how to define ops in Rust and then call them from
+//!  JavaScript.
+// https://github.com/denoland/deno/blob/main/core/examples/hello_world.rs
+// https://github.com/denoland/deno/blob/main/core/examples/http_bench_json_ops.rs
```

Comment:
These examples were useful for understanding the runtime. We would need to add some "extensions" to the runtime to execute external rust functions